test/azul_test_case.py

```diff
@@ -56,6 +59,9 @@ class AzulTestCase(TestCase):
     def setUpClass(cls) -> None:
         super().setUpClass()
 
+        cls.aws_account_id = config.aws_account_id
+        os.environ['AZUL_AWS_ACCOUNT_ID'] = '123456789012'
```

Comment:
You should mention in a comment where the value comes from or, even better, use the corresponding moto constant here.